src/robot/variables/evaluation.py

```diff
@@ -149,6 +149,8 @@ def _import_module(self, name):
         if name in PYTHON_BUILTINS:
             raise KeyError
         try:
+            if hasattr(builtins,name):
+                return getattr(builtins, name)
```

Comment:
There's already a check for the `name` possibly being a built-in few lines above. The reason it doesn't work is that it uses the `PYTHON_BUILTINS` module level attribute that's not updated if `builtins` is changed. That check could just be replaced with `if hasattr(builtins, name)` and the whole constant removed. The constant feels unnecessary in general and I believe it has only been needed when we supported Python 2 and Python 3.